### PR TITLE
Fix CI (partially)

### PR DESCRIPTION
**Pull request by sliptype**
### Description

Partially fix CI breakages due to npm workspaces. Still a couple outstanding issues including building storybook and the codepush steps

### CODE REVIEW

File: .circleci/src/commands/@web-commands.yml [hunk 1]
```diff
@@ -93,6 +93,12 @@ web-distribute:
     - checkout
     - attach_workspace:
         at: ./
+    - run:
+        name: Install dependencies
+        command: rm -rf node_modules && SKIP_POD_INSTALL=true npm ci
```
Comment (dylanjeffers) [inline .circleci/src/commands/@web-commands.yml:98]: just curious, we couldn't use the cached version from the init phase?

File: .circleci/src/jobs/@stems-jobs.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ stems-init:
     - persist_to_workspace:
         root: ./
         paths:
-          - packages/node_modules
+          - node_modules
```
Comment (dylanjeffers) [inline .circleci/src/jobs/@stems-jobs.yml:26]: nice

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

File: packages/mobile/android/app/build.gradle [hunk 3]
```diff
@@ -58,7 +58,7 @@ project.ext.envConfigFiles = [
 ]
 
 apply from: "../../../../node_modules/react-native-code-push/android/codepush.gradle"
-apply from: "../../node_modules/@sentry/react-native/sentry.gradle"
+apply from: "../../../../node_modules/@sentry/react-native/sentry.gradle"
```
Comment (dylanjeffers) [inline packages/mobile/android/app/build.gradle:61]: nice

File: scripts/postinstall.sh [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
-# Symlink react-native into the mobile package bc npm doesn't
-# support nohoist
-cd packages/mobile/node_modules
-ln -s ../../../node_modules/react-native react-native
-cd ../ios
-if command -v pod >/dev/null; then
-  pod install
-fi
+
+if [[ -z "${SKIP_POD_INSTALL}" ]]; then
```
Comment (dylanjeffers) [inline scripts/postinstall.sh:6]: sneaky, makes sense tho

**Review by dylanjeffers — APPROVED** [review]
the lords work

File: .circleci/src/commands/@web-commands.yml (hunk 1 above)
Comment (sliptype) [reply .circleci/src/commands/@web-commands.yml:98]: Actually a very crazy issue that I don't quite understand. Electron builder always fails doing symlinks from the node_gyp artifacts with a `Read Only File System` error. This is stuff that just exists in the node_modules folder. Only those node_gyp artifacts (like `python3` for example) throw this error when you try to modify them. But, making a copy of the file and renaming it to python3 fixes everything. Reinstalling node modules has the same effect. Maybe it has something to do with node_gyp compiling dependencies for one platform in the CI init step and then trying to consume them on a different platform (the mac os circle ci image). But idk how the change in dependency structure would have caused that

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sk-ci-fixes

File: .circleci/src/commands/@web-commands.yml (hunk 1 above)
Comment (dylanjeffers) [reply .circleci/src/commands/@web-commands.yml:98]: I see what you mean about maybe compiling for different platforms but to your point it's weird that it worked before. Well something to note for monorepo

**Pull request merged**